src/app/ConfigureUtilities.jsx

```diff
@@ -52,43 +31,6 @@ const updateSelectedItem = (parentMenuItem, selectedItem) => {
 
 class ConfigureUtilties {
```

Comment:
This doesn't break if we don't have Theme, Locale or Bidi utilities does it? I notice the update selected items will reference all three

Reply:
no it's all good.